src/tree_walker.js

```diff
@@ -36,7 +36,7 @@ function TreeWalker(node) {
   this.currentNode = node;
 
   /**
-   * @const {!Document}
+   * @const {Document}
```

Comment:
I think this was resolved by https://github.com/google/incremental-dom/commit/53dfbf017d6caa230ccd59aabeee5c83800925da? If so, `{!Document}`.